api/db-migrations/000012_add_traffic_rule_name.up.sql

```diff
@@ -0,0 +1,14 @@
+UPDATE router_versions
```

Comment:
Missing semicolon at the end. But more importantly, are you sure this query is doing what it should? From what I can tell, this will update every row with the same result.

Could you help verify this and capture the results for a couple of rows, either here in the comments or in the PR description so it's easier to be on the same page?

Reply:
Given current version = 11 and dirty = false `schema_migrations` table,

`Before`
![image](https://user-images.githubusercontent.com/25025366/187351364-3c68dbf4-197b-4762-a61c-d41fa77ef83c.png)

`After`
![image](https://user-images.githubusercontent.com/25025366/187351403-a62ea498-367d-482b-9e3a-f3bb7badd2eb.png)

By every row, do you mean for each router version?

Reply:
> By every row, do you mean for each router version?

Yes, 2 different router versions (each with multiple different rules, as you've captured above).

Reply:
Ah, I think I got what you mean. I've updated the query now and the following screenshots depict changes to DB data after before and after app start-up with schema_migrations `version=11, dirty='f'`.

`Before`
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/25025366/187590101-86971cd6-4f5e-43ac-a274-5e8f0a0e96b8.png">

`After`
<img width="1162" alt="image" src="https://user-images.githubusercontent.com/25025366/187590335-44ec0a39-12cc-4191-b32d-84bee9c7275a.png">

Reply:
The output does look right to me, but strangely, when I tried to run it, I got the following error: `cannot extract elements from a scalar`. I dumped our internal dev DB to test the migration. Could you check this further?

Reply:
I think I figured out the problem - the `traffic_rules` column does not have a NOT NULL constraint and the null values are affecting the result. Could you update the query to handle such rows?

Reply:
The issue is with "null" strings rather than null values, I've updated the query to handle these rows. Also tested them on internal dev and staging DB data.